mozilla/bedrock
# [bug 1343823] Remove old /whatsnew templates

## Description
- Removes a boat load of old `/whatsnew` pages.
- The `/whatsnew` template for firefox 54 (mobile promo) becomes the new default for all fallback URLs.
- Renames `firefox/whatsnew_50.lang` to `firefox/whatsnew.lang`.
- Adds functional tests for default `/whatsnew` page.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1343823

## Testing
Demo: https://www-demo3.allizom.org/firefox/whatsnew/

- Check default `/whatsnew` and check for any regressions?
- Make sure I didn't miss removing any supreflous assets, or did remove anything that's still needed?
